### PR TITLE
Minor changes in Czech translation

### PR DESCRIPTION
**Pull request by Poggicek**

### CODE REVIEW

**Comment by Poggicek** [comment]
Gaybarra using translations from google

**Comment by Trsak** [comment]
Well this commit contains many spelling mistakes

File: locale/cs.json [hunk 1]
```diff
@@ -4,19 +4,19 @@
         "humanizer_language": "cs"
     },
     "restarter": {
-        "start_timeout": "server se nestihl spustit v požadovaném čas",
+        "start_timeout": "server se nestihl spustit v požadovaném čase",
         "crash_detected": "pád detekován",
-        "partial_crash_warn": "Kvůli částečné havárii dojde za 1 minutu k restartu toho serveru. Prosím odpojte se nyní.",
-        "partial_crash_warn_discord": "Kvůli částečné havárii se **%{servername}** restartuje za 1 minutu.",
+        "partial_crash_warn": "Kvůli částečnýmu pádu dojde za 1 minutu k restartu tohoto serveru. Prosím odpojte se nyní.",
```
Comment (Trsak) [inline locale/cs.json:9]: Should be "částečnému"

File: locale/cs.json [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
+        "partial_crash_warn_discord": "Kvůli částečnýmu pádu se **%{servername}** restartuje za 1 minutu.",
```
Comment (Trsak) [inline locale/cs.json:10]: Should be "částečnému"

File: locale/cs.json [hunk 3]
```diff
@@ -32,6 +32,6 @@
     "nui_warning": {
         "title": "VAROVÁNÍ",
         "warned_by": "Varován od:",
-        "instruction": "Drž [MEZERNÍK] po dobu 10 sekund pro zavření této zprávy."
+        "instruction": "Drž [MEZERNÍK] po dobu 10-ti sekund pro zavření této zprávy."
```
Comment (Trsak) [inline locale/cs.json:35]: Should be just "10"... What the hell is "10-ti"?

Comment (czsquizer) [reply locale/cs.json:35]: You can use 10-ti or 10 to explain the same meaning of the word. (10 prsty a 10-ti prsty is the same and both are grammatically correct) :wink:

Comment (Trsak) [reply locale/cs.json:35]: @czsquizer No you can not. Time to learn grammar man! https://www.meriva-preklady.cz/clanek/cesky-pravopis-10lety-18lety-pridavnejmeno_65

Comment (czsquizer) [reply locale/cs.json:35]: Mhmhmm, I got wrong :joy: sorry. Just confused it with another thing.

**Comment by tabarra** [comment]
Hey @Trsak @czsquizer @Poggicek can you guys figure this out real fast?  
I need to push the update in a few hours :p

**Comment by czsquizer** [comment]
The Trsaks version is correct, wothout spelling mistakes.

**Comment by Trsak** [comment]
I would send PR, but since i did PR #226 on main branch of my fork, I would prefer you to resolve it first :D

File: locale/cs.json (hunk 3 above)
Comment (Poggicek) [reply locale/cs.json:35]: Yeah I was wrong. Trsak's version is correct

**Comment by Poggicek** [comment]
Fixed in #247

**Pull request merged**